libmscore/text.cpp

```diff
@@ -2659,7 +2659,12 @@ Element* Text::drop(const DropData& data)
                         }
                   else {
                         startEdit(data.view, data.pos);
-                        curLine().insert(_cursor, QChar(code));
+                        if (QChar::requiresSurrogates(code)) {
```

Comment:
ok, well good that the code is fixed, in case it gets exercised in the future.  I would say we should make it possible to bring up text palette when not in edit mode and drop onto text elements when not in edit mode, unless there is a compelling reason to prohibit it.